.github/workflows/release.yml

```diff
@@ -0,0 +1,62 @@
+name: Release chart package
+
+on:
+  push:
+    tags:
+      - '*'
+  # TODO delete for production purposes
+  pull_request:
+    branches:
+      - main
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  docker:
```

Comment:
Think this should be changed, right?

Reply:
Yes, sue, will change that, it's pretty much draft for now to test pipeline